linux_os/guide/system/accounts/accounts-pam/enable_authselect/tests/empty.fail.sh

```diff
@@ -0,0 +1,6 @@
+# platform = multi_platform_rhel,multi_platform_fedora
+# remediation = none
+
+rm -rf /etc/authselect/*
+touch /etc/authselect/authselect.conf
```

Comment:
I manually installed a RHEL8 and chose minimal install.
The installed system didn't have file authselect configued and file `/etc/authselect/authselect.conf` did not exist.
![image](https://user-images.githubusercontent.com/7460169/155142584-cd183157-f274-46ed-b8dd-3d4bb97a2651.png)

Does it make sense to add a test where only `/etc/authselect/authselect.conf` is missing?

Reply:
At least in my researches I didn't find a way to "disable" `authselect` once a profile is selected unless the symbolic links are manually removed. If this is the case, the system could still have a content in `/etc/authselect/authselect.conf` but the authselect profile are not actually in use.

I would recommend to assess if an authselect profile is selected by testing if the PAM files are symlinks to /etc/authselect

Reply:
Done, plus the remediation now remediates with `--force` if the RPM check of the `pam` package discovers no changes to the package - it is able to detect changes to config files. So if there are no changes, there is no reason why not to force.